src/pages/plans/splitPlans/UpperBody.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { exercises } from '../../../data/exercises';
+
+const UpperBody = () => {
+  const relatedExercises = exercises.filter(
```

Comment:
If HTML content in all tabs will be the same, you can use one map function and change it's data instead of 4 different files with the same structure.